packages/core/package.json

```diff
@@ -23,7 +23,6 @@
   "devDependencies": {
     "@eth-optimism/smock": "^1.1.9",
     "@awaitjs/express": "^0.3.0",
-    "@eth-optimism/smock": "^1.1.9",
```

Comment:
Why remove?

Reply:
it was duplicated in this package.json :)